fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/Election.java

```diff
@@ -12,15 +14,10 @@
     private long creation;
     private long start;
     private long end;
-    private boolean writeIn;
-    private String question;
-    private List<String> ballotOptions;
-
-    //votes as attribute ?
-
 
+    private List<ElectionQuestion> electionQuestions;
     public Election() {
-        this.ballotOptions = new ArrayList<>();
+      this.electionQuestions = new ArrayList<>();
```

Comment:
I understand that you want to allow multiple questions in an election, right? The problem here is that you have an election with only a list of questions, deleting all the other important information (ballot options, write in and the actual question). This means that in `LaoRepository` when you handle the ElectionSetup message you are ignoring this information. A solution for this is to create a separate model for questions with this information or keep the old implementation with only one question

Reply:
The ElectionQuestion object has all the fields that I deleted in the Election class. Then all the relevant information are kept together by question in an object instead of having lists of lists in Election.

Reply:
I must have missed the fact that the list of questions was a list of `ElectionQuestion` and not String, my bad. Indeed the implementation is ok and there's nothing to change here